test/suite/test_inmem01.py

```diff
@@ -88,6 +88,7 @@ def test_insert_over_delete(self):
 
     # Run queries after adding, removing and re-inserting data.
     # Try out keeping a cursor open while adding new data.
+    @unittest.skip("Temporarily disabled")
```

Comment:
Unless the fix is definitely going to be merged in today, I'd rather disable it. I don't like having a red build as "expected" since it increases the chance of someone mistakenly merging in a regression.

Reply:
Just leave a comment in ticket opened for fixing the `inmem` test so that we don't forget to enable it again.